### ISSUE TITLE
Fails to parse Camera on update to UVC 3.10.0

### ISSUE THREAD

**Issue by wralb** [mzac/unifi-video-mqtt#5]
Thanks for a great simple solution to generating Unifi camera motion snapshots.  Worked flawlessly for me for months.

I just upgraded to unifi video 3.10.0 and the unifi-video-mqtt.sh script stopped doing what it should.  A quick inspection shows that Ubiquiti added an extra field to motion.log, which breaks the line which extracts the camera ID from the log line.  I got it to work again by updating the line which extracts LAST_CAM variable to be:

<code>
LAST_CAM=`echo $LAST_MESSAGE | awk -F 'Camera[][]' '{print $2}'`
</code>

Basically, I changed "[" to "Camera[" in the awk command.

There may be a better way to fix this, I'm no expert.

**Comment by mzac** [comment]
Thanks, it would be good if we could find a way to check the version of unifi video running or, we run a grep to see what format the file is in then we decide which way to grep it.  I'll take a look at it soon.

If you figure out a good way to do the above feel free to create a PR

**Comment by mzac** [comment]
Try 5bbcc012b0d565789967127bbc0ea409ab84089b and see if it works for you
